rust/operator-binary/src/catalog/config.rs

```diff
@@ -104,6 +104,11 @@ impl CatalogConfig {
                     .to_catalog_config(&catalog_name, catalog_namespace, client)
                     .await
             }
+            TrinoCatalogConnector::GoogleSheet(connector) => {
```

Comment:
Just curious: would there be any advantage in having a trait for `to_catalog_config`, so the `match` statement wouldn't have to changed every time a new enum variant is added? (I like the enum pattern but am not deep enough into Rust to know the pros/cons).

Reply:
That's fine by me